src/render/PanelHeaderProvider.js

```diff
@@ -79,6 +83,20 @@ export const PanelHeaderProvider = {
   },
 
   getTypeLabel: (element) => {
+
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    const elementTemplates = useService('elementTemplates', false);
+
+    if (!elementTemplates) {
+      return;
```

Comment:
Can you explain why we not continue if there was no `elementTemplates` services? Should we not also then follow up with https://tasks.bpmn.io/board?c=%21Inbox&s=label%3A%22spring+cleaning%22+ and following?

(I know there is an existing test, that covers this and succeeds, I still don't understand why :-) )

Reply:
My assumption: If you use a properties panel without `elementTemplates` you also don't have elements that have element templates applied.

Reply:
How would that even work?

Reply:
Ah sorry, I see what I missed there. This is just broken 😄

Reply:
You are absolutely right, it doesn't make sense to stop there if no `elementTemplates` service is available. The test that should verify it still retrieves the element type was just broken, the mocked service was injected in any case.

I fixed that via https://github.com/bpmn-io/bpmn-js-properties-panel/pull/627/commits/11c14257d249c7a4e2f8a1035e64cdadfb5992b7